airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import { faGripLinesVertical } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import { ReflexContainer, ReflexElement, ReflexSplitter } from "react-reflex";
+import styled from "styled-components";
+
+import { useDocumentationPanelContext } from "./ConnectorDocumentationContext";
+
+const PanelGrabber = styled.div`
+  height: 100vh;
+  padding: 6px;
+  display: flex;
+`;
+
+const GrabberHandle = styled(FontAwesomeIcon)`
+  margin: auto;
+  height: 25px;
+  color: ${({ theme }) => theme.greyColor20};
+`;
+
+//NOTE: ReflexElement will not load its contents if wrapped in an empty jsx tag along with ReflexSplitter.  They must be evaluated/rendered separately.
+
+export const ConnectorDocumentationLayout = ({ children }: { children: [React.ReactNode, React.ReactNode] }) => {
+  const [left, right] = children;
+
+  const { documentationPanelOpen } = useDocumentationPanelContext();
+
+  return (
+    <ReflexContainer orientation="vertical" windowResizeAware={true}>
+      <ReflexElement className="left-pane"> {left}</ReflexElement>
```

Comment:
Unnecessary space next to `{left}`

Reply:
In testing, the left side may need more wrapping, such a min-width that overflows if the resizing is bigger so that it doesn't always resize the left side based on the size of the content but there's a limit to it.

Reply:
I went with a slightly different solution moving closer to Nico's designs.  Let me know your thoughts there. 

https://user-images.githubusercontent.com/63751206/167896932-bd7eee19-6586-4c07-b96c-9f81e42931f2.mov

Things we could do in addition:
- Add an "x" when the Setup Guide is in that minimal bar view
- Have the Setup Guide panel "snap" to the 80px size as soon as we hit the 350px break point